src/patternfly/components/Page/page.scss

```diff
@@ -415,7 +421,6 @@ $pf-page-v6--height-breakpoint-map: build-breakpoint-map("base", "sm", "md", "lg
 .#{$page}__main-section,
 .#{$page}__main-group,
 .#{$page}__main-wizard {
-  &:last-child,
   &:only-child,
```

Comment:
Sorry didn't catch this before, but I'm thinking we probably don't want `:only-child` to fill, just like `:last-child` - you'd need to apply `.pf-m-fill` manually if you want something to fill. wdyt @srambach?

Reply:
I agree. Updated it to remove only-child as well.